ui/webui/src/components/AnacondaWizard.jsx

```diff
@@ -55,14 +59,16 @@ const getSteps = ({
                 <Title headingLevel="h2" size="xl">
                     {label}
                 </Title>
-                {stepNotification &&
-                 (stepNotification.step === currentStepId) &&
-                 <Alert
-                   isInline
-                   title={stepNotification.message}
-                   variant="danger"
-                 />}
-                {children}
+                <TextContent>
+                    {stepNotification &&
+                     (stepNotification.step === currentStepId) &&
+                     <Alert
+                       isInline
+                       title={stepNotification.message}
+                       variant="danger"
+                     />}
+                    {children}
+                </TextContent>
```

Comment:
I think you should have instead wrapped only the `Alert` in `TextContent`, but it probably works this way too.